pkgs/tools/security/gnupg/minimal.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, gnupg }:
+
+let 
+  p = gnupg.override {
+    pcsclite = null;
+    sqlite = null;
+    pinentry = null;
+    guiSupport = false;
+    adns = null;
+    gnutls = null;
+    libusb = null;
+    openldap = null;
+    readline = null;
+    zlib = null;
+    bzip2 = null;
+  };
+in
+  stdenv.mkDerivation rec {
+    pname = "gnupg-minimal";
+    version = p.version;
+
+    src = p.src;
+
+    nativeBuildInputs = p.nativeBuildInputs;
+    buildInputs = p.buildInputs;
+
+    configureFlags = p.configureFlags ++ [ "--disable-sqlite" ];
```

Comment:
please add this to the main gnupg derivation, like it's done with pcsclite.

Reply:
Will do